### DISCUSSION TITLE
Changelog and/or queue visibility for extension builds?

### DISCUSSION THREAD

**Discussion by marklit** [duckdb/duckdb#8187] [General]
I'm working off the main branch a lot and after recompilation, I'm often presented with something along the line of the following:

```sql
INSTALL inet;
```

```
Error: HTTP Error: Failed to download extension "inet" at URL "http://extensions.duckdb.org/cc60796b7a/linux_amd64/inet.duckdb_extension.gz"
Extension "inet" is an existing extension.

Are you using a development build? In this case, extensions might not (yet) be uploaded.
```

Is there somewhere that lists the progress of building these extensions and how far back in the queue any one extension is at any given time? It would save needing to periodically re-run the `INSTALL` command while waiting for the new extension to turn up.

Also, I think the `http` in the error message is a mistake as the domain appears to point to AWS Cloudfront and uses HTTPS.

**Comment by Mause** [comment]
Hey there! If you want to look at the progress of extensions, you can have a look at the actions for your platform/distribution: https://github.com/duckdb/duckdb/actions/workflows/LinuxRelease.yml?query=event%3Arepository_dispatch

The `http` is not a mistake actually - OpenSSL is a pretty heavy dependency, so it isn't included in the core (only in the httpfs extension). Instead, we sign the extensions using our private key and embed the public key in the core, and verify extensions at load time

**Comment by marklit** [comment]
That's what I was looking for. Thank you.
